src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A duke.Task object with a deadline
+ */
+public class Deadline extends Task{
+    protected String by;
+
+    /**
+     * Create task based on description and deadline
+     *
+     * @param description The description of the task
```

Comment:
I noticed that your parameter descriptions were missing punctuation behind them throughout your code, what do you think of adding them?